cmd/fluxd/main.go

```diff
@@ -345,8 +347,27 @@ func main() {
 	}
 
 	// Import GPG keys, if we've been told where to look for them
+
+	if err != nil {
+		logger.Log("error", err.Error())
+	}
+
+	// default to no signature verification
+	gitVerifySignaturesMode, err := fluxsync.ToVerifySignaturesMode(fluxsync.VerifySignaturesModeNone)
+
+	if err != nil {
+		logger.Log("error", err.Error())
+	}
+	if *gitVerifySignaturesModeStr != fluxsync.VerifySignaturesModeDefault {
```

Comment:
`pflag` offers a `Changed` method, using this method you can check if a flag has been modified from its default value. Using this flag this could be reduced to:

```go
if fs.Changed("git-verify-signatures") && !fs.Changed("git-verify-signatures-mode") {
  gitVerifySignaturesMode = fluxsync.VerifySignaturesModeAll
}
```

In addition, can you please move this a bit higher up (`L307`) below the other backwards maintaining `if`-statement?